textworld/generator/chaining.py

```diff
@@ -124,7 +131,8 @@ def get_rules(self, depth: int) -> Iterable[Rule]:
         if depth < len(self.rules_per_depth):
             return self.rules_per_depth[depth]
         else:
-            return self.logic.rules.values()
+            # Examine, look and inventory shouldn't be used for chaining.
+            return self.kb.rules.get_matching("^(?!(examine.*|look.*|inventory.*)).*")
```

Comment:
Can the final `.*` be replaced with `$`?

Reply:
I decide to change the `get_matching` method to accept a `exclude` argument. See https://github.com/Microsoft/TextWorld/pull/81/files#diff-beb079c3279f087d11de37962f4e620dR123